prime/src/components/borrow/actions/UniswapSwapActionCard.tsx

```diff
@@ -59,29 +59,27 @@ export default function UniswapSwapActionCard(props: ActionCardProps) {
   const amountInExact = userInputFields?.at(0) ?? '';
   const amountOutMin = userInputFields?.at(1) ?? '';
   const tokenTypeIn = (userInputFields?.at(2) ?? TokenType.ASSET0) as TokenType;
-  const slippage = userInputFields?.at(3) ?? DEFAULT_SLIPPAGE_PERCENTAGE;
+  const slippage = userInputFields?.at(3) ?? '';
```

Comment:
What happened to default slippage?

Reply:
Resolved offline. It handles it on it's own.